include/cppast/compile_config.hpp

```diff
@@ -22,10 +22,10 @@ enum class cpp_standard
     cpp_03,
     cpp_11,
     cpp_14,
-
     cpp_1z, //< Upcoming C++17 (experimental).
+    cpp_20,
 
-    cpp_latest = cpp_standard::cpp_14, //< The latest supported C++ standard.
+    cpp_latest = cpp_standard::cpp_20, //< The latest supported C++ standard.
```

Comment:
Note there is no C++17 or 2A.

Reply:
Can you add them as well for completeness?